diff --git a/make/common/JavaCompilation.gmk b/make/common/JavaCompilation.gmk
index f48aefcd5..00f2f4c21 100644
--- a/make/common/JavaCompilation.gmk
+++ b/make/common/JavaCompilation.gmk
@@ -268,7 +268,8 @@ define SetupJavaCompilationBody
   JAVA_WARNINGS_ARE_ERRORS ?= -Werror
 
   # Tell javac to do exactly as told and no more
-  PARANOIA_FLAGS := -implicit:none -Xprefer:source -XDignore.symbol.file=true -encoding ascii
+  # AprismJDK: use utf-8 instead of ascii to support Unicode in comments/javadoc
+  PARANOIA_FLAGS := -implicit:none -Xprefer:source -XDignore.symbol.file=true -encoding utf-8
 
   $1_FLAGS += -g -Xlint:all $$($1_TARGET_RELEASE) $$(PARANOIA_FLAGS) $$(JAVA_WARNINGS_ARE_ERRORS)
   $1_FLAGS += $$($1_JAVAC_FLAGS)
